src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -237,21 +275,32 @@ def __init__(self):
             self._jm = JobManager()
         if self._msg_map is None:
             self._msg_map = {
-                STATUS_ALL: self._lookup_all_job_states,
+                CANCEL: self._cancel_jobs,
                 CELL_JOB_STATUS: self._lookup_job_states_by_cell_id,
-                STATUS: self._lookup_job_states,
-                STATUS_BATCH: self._lookup_job_states_batch,
                 INFO: self._lookup_job_info,
-                INFO_BATCH: self._lookup_job_info_batch,
+                INFO_BATCH: self._lookup_job_info,
+                LOGS: self._get_job_logs,
+                RETRY: self._retry_jobs,
                 START_UPDATE: self._modify_job_updates,
+                START_UPDATE_BATCH: self._modify_job_updates,
+                STATUS: self._lookup_job_states,
+                STATUS_BATCH: self._lookup_job_states,
+                STATUS_ALL: self._lookup_all_job_states,
                 STOP_UPDATE: self._modify_job_updates,
-                START_UPDATE_BATCH: self._modify_job_updates_batch,
-                STOP_UPDATE_BATCH: self._modify_job_updates_batch,
-                CANCEL: self._cancel_jobs,
-                RETRY: self._retry_jobs,
-                LOGS: self._get_job_logs,
+                STOP_UPDATE_BATCH: self._modify_job_updates,
             }
 
+    def _get_job_ids(self, req: JobRequest = None):
```

Comment:
More flexible way to get job data out of a job request -- no need for a separate function for the batch version of a request.

(batch versions to be removed in a subsequent PR)